artifactory/utils/repositoryutils.go

```diff
@@ -16,13 +17,15 @@ const (
 	REMOTE
 	VIRTUAL
 	FEDERATED
+	RELEASE_BUNDLES
```

Comment:
Go's conversion for consts is came-case and not snake case. Let's convert all the above consts.